linux_kernel_modules/rtc_sync/rtc_sync.c

```diff
@@ -75,11 +77,11 @@ static int rtc_hctosys(void)
 	return 0;
 }
 
-//systohc sets RTC with system time
+/*systohc sets RTC with system time*/
+
```

Comment:
Don't put a blank line after the comment and before the function.  It makes it less clear that the comment is attached to the function.